exporter/opentelemetry-exporter-zipkin-json/tests/encoder/test_v2_json.py

```diff
@@ -33,6 +34,14 @@
 
 # pylint: disable=protected-access
 class TestV2JsonEncoder(CommonEncoderTestCases.CommonJsonEncoderTest):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Tests should not interfere with each other (running them in randomized order is precisely how we try to catch any interferences). `setUpClass` should be a possible option for any suite of test cases if the tester prefers it.

Reply:
Agreed @ocelotl. Tests should be separate and non-conflicting, such that any individual test could be run by itself. 
Otherwise, one would have to run preliminary tests prior to running any desired test.

Reply:
Running tests randomly is different from the ability to use `setUpClass`. I am not against the idea of having the `setUpClass` but the author is using it for setting up the `TracerProvider` (without other trace pipeline steps? is it even effective since the exporter tests are failing?). The `TracerProvider` and `MeterProvider` hold state which could potentially introduce side effects (ex an instrument with same name cannot be created in different tests) which is why we discourage to use `setUpClass` for initializing the trace/metric/log provider pipeline.

Reply:
What I mean is that we do not forbid the testers from using `setUpClass` as a general rule. If there is a valid use case for it and it serves testing purposes, then it can be used.